src/openms/include/OpenMS/FORMAT/MSstatsFile.h

```diff
@@ -82,14 +82,34 @@ namespace OpenMS
                       const String& condition,
                       const String& mixture,
                       const String& retention_time_summarization_method);
-    
+
     private:
-     
+
+        typedef OpenMS::Peak2D::IntensityType Intensity;
+        typedef OpenMS::Peak2D::CoordinateType Coordinate;
+
         const String na_string = "NA";
 
+
         /*
-        *  @brief: Internal function to check if MSstats_BioReplicate and MSstats_Condition in Experimental Design
-        */
+         *  @brief: Struct to aggregate intermediate information from ConsensusFeature and ConsensusMap,
+         *  such as filenames, intensities, retention times, labes and features (for further processing)
+         */
+        struct AggregatedConsensusInfo
+        {
+          std::vector< std::vector< String > > consensus_feature_filenames;           // Filenames of ConsensusFeature
```

Comment:
use proper doxygen commnts //<

Reply:
add information what the entries correspond to: e.g. consensus_feature_filenames[0] corresponds to the the filenames stored in the first ConsensusMap